userguide/storage.rst

```diff
@@ -1767,11 +1767,11 @@ in :ref:`Encryption` **before** attempting to replace the failed
 drive. Then, follow the steps 1 and 2 as described above. During step
 3, a prompt will appear to input and confirm the passphrase for the
 pool. Enter this information then click :guilabel:`Replace Disk`.
-Immediately
-:ref:`restore the encryption keys to the pool<Managing Encrypted Volumes>`.
 
-.. warning:: Access to the pool will be permanently lost unless the
-   encryption keys are restored to the pool before the next system reboot!
+Wait until the resilvering is complete before
+:ref:`restoring the encryption keys to the pool<Managing Encrypted Volumes>`.
```

Comment:
Have you built this?  I'm not sure if the space between the link text and the URL is required or not.